docs/general_config.md

```diff
@@ -298,6 +298,7 @@ These flags help control the behavior of click-to-photon monitoring in applicati
 | Parameter Name        |Units                  | Description                                                                        |
 |-----------------------|-----------------------|------------------------------------------------------------------------------------|
 |`targetHealthColors`   |[`Color3`, `Color3`]   | The max/min health colors for the target as an array of [`max color`, `min color`], if you do not want the target to change color as its health drops, set these values both to the same color                                              |
+|`showRefTarget`        |`bool`                 | Show a reference target to re-center the view between trials/sessions?             |
```

Comment:
Given that the other parameters are `referenceTargetColor` and `referenceTargetSize`, I believe this feature would work better being called `showReferenceTarget`.